src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,26 +4,26 @@
 
 public class Diamond {
 
-	public String print(int n) {
```

Comment:
maybe this is not a suitable name since there is nothing printed in the method, but rather created?